windows-release/azure-pipelines.yml

```diff
@@ -49,6 +49,18 @@ parameters:
   displayName: "Include free-threaded builds (3.13 and later)"
   type: boolean
   default: true
+- name: DoJIT
+  displayName: "Build the JIT compiler (3.14 and later)"
+  type: boolean
+  default: false
+- name: DoJITEnabled
+  displayName: "Enable the JIT compiler by default (not used yet)"
+  type: boolean
+  default: false
+- name: DoJITFreethreaded
+  displayName: "Build the JIT compiler for free-threaded builds (not used yet)"
+  type: boolean
+  default: false
```

Comment:
In terms of ordering, use descending version numbers, and put the future proofing ones right at the end. Having the version numbers in there is very important or we'll definitely get the releases wrong :)

If it makes more sense, you can make a drop down with `type: string` and `values:` like the signing certificates above. Trick here is that you can't select "none" unless it's actually in the list, and you'll need `eq()` checks (unless you can figure out a way to use the option directly and have better display text, which may be possible, but I don't know how).